### PR TITLE
Fixed french translation issues for partitoning guide

### PR DESCRIPTION
**Pull request by JayBeeDe**
Error messages should remain in English because they come directly from the backend.

### CODE REVIEW

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
{{.cds.build.pelicanoutput}}
```

File: pages/cloud/dedicated/partitioning_ovh/guide.fr-ca.md [hunk 1]
```diff
@@ -192,24 +192,24 @@ Le tableau suivant donne une vue d'ensemble de la compatibilité des systèmes d
...
+⁴ L'installateur ESXi ne prend pas en charge les schémas de partitionnement personnalisés. Le partitionnement est défini par l'éditeur du logiciel. Néanmoins, l’[API OVHcloud](https://ca.api.ovh.com/) peut vous donner une idée de ce à quoi ressemble le partitionnement : pour plus d'informations, consultez [les templates OVHcloud](#OVHcloudtemplates).<br />
 
 > [!warning]
 >
-> Ce tableau est fourni uniquement à titre d'information. À noter que la compatibilité LVM et surtout filesystem dépend également de l’OS (template OVHcloud) installé. Consultez les [templates OVHcloud](#OVHcloudtemplates) pour plus de détails.
+> Ce tableau est fourni uniquement à titre d'information. À noter que la compatibilité LVM et surtout le système de fichiers dépend également de l’OS (template OVHcloud) installé. Consultez les [templates OVHcloud](#OVHcloudtemplates) pour plus de détails.
```
Comment (OVH-Team-Guides) [inline pages/cloud/dedicated/partitioning_ovh/guide.fr-ca.md]: À noter que la compatibilité LVM et surtout le système de fichiers **dépendent** également de l’OS (template OVHcloud) installé.

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
{{.cds.build.pelicanoutput}}
```

**Comment by OVH-Team-Guides** [comment]
Thanks for your contribution @JayBeeDe

**Pull request merged**